[Docs]: Simplify code creation logic for Code and CodeInput docs

ArtemKhvorostianyi opened the pull request:
<img width="922" height="534" alt="Screenshot 2025-10-27 at 14 25 26" src="https://github.com/user-attachments/assets/438ecb34-07c7-43e1-8140-2c4e1ceeb65a" />

Docs -> Code
Docs -> Code Input

Avoided Layouts usage for simple code blocks
Maybe we have another issue (I'm not sure that this is an issue)
Width(Size.Full) doesn't actually take all the width of the box

<img width="903" height="389" alt="Screenshot 2025-10-27 at 14 26 59" src="https://github.com/user-attachments/assets/01e4e615-ff49-443b-9ffd-82a8b23c68bb" />
<img width="1305" height="786" alt="Screenshot 2025-10-27 at 14 28 04" src="https://github.com/user-attachments/assets/4993208d-5d2b-40a7-ad17-2273d1331b92" />
<img width="1437" height="784" alt="Screenshot 2025-10-27 at 14 28 14" src="https://github.com/user-attachments/assets/3325a27c-e422-45e4-a5cc-92838a943f6c" />
<img width="1438" height="796" alt="Screenshot 2025-10-27 at 14 28 21" src="https://github.com/user-attachments/assets/b1e39724-8c1e-4e6b-a38b-ddac432bc67c" />
<img width="1437" height="790" alt="Screenshot 2025-10-27 at 14 28 29" src="https://github.com/user-attachments/assets/6abea7af-6456-42f9-84b3-05d1d59622b3" />
<img width="1440" height="673" alt="Screenshot 2025-10-27 at 14 28 40" src="https://github.com/user-attachments/assets/85868de0-ea7a-4670-82c3-16289dd36831" />

rorychatt commented:
hmmm.... looks kind of bad with the boxes not taking full area.

What do you think @ArtemKhvorostianyi ?

Maybe this is a bug? Shouldn't elements take full width/height of their container, unless it's specified?

or maybe we can just return a code inside of a layout. idk

ArtemKhvorostianyi commented:
1. Renamed Code input doc file to actually CodeInput.md
2. Used UseState("some code").ToCodeInput() for code Input doc
3. refactored code visibility in code(avoided \n usage)
